cypress/page-objects/lib/ui-element.ts

```diff
@@ -6,21 +8,28 @@ export class UIElement {
     this._selector = selector
     this.isTextSelector = isTextSelector
   }
-
-  contains() {
-    return cy.contains(this._selector)
+  public get element() {
+    return cy.get(this._selector)
   }
 
-  protected getElement() {
+  getElement() {
     return this.isTextSelector
       ? cy.contains(this._selector)
-      : cy.get(this._selector)
+      : cy.get(this._selector, { timeout: Timeouts.FifteenSecondsTimeout })
+  }
+
+  contains(text: string) {
+    return this.getElement().contains(text)
   }
 
   click() {
     this.getElement().click()
   }
 
+  forceClick() {
```

Comment:
No need in this method, you can just pass parameters into `click` method and that's it.